src/Microsoft.DotNet.Tools.Pack/NuGet/PathResolver.cs

```diff
@@ -0,0 +1,301 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace NuGet
+{
+    public static class PathResolver
```

Comment:
At first I had left it out entirely and thrown on any wildcard in the path.  We don't need it for our scenarios.  If you'd prefer I can do that here.